### PR TITLE
feat: webpack-dev-server is used for develop

### PR DESCRIPTION
**Pull request by ValeraS**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [ ] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.

- [ ] Closes currently open issue: Closes #XXX

### CODE REVIEW

**Comment by tbushman** [comment]
@ValeraS It's working for me. Thanks and sorry for the trouble. @tchaffee Do you want to merge this or I will?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -318,7 +315,7 @@ resources you need (e.g. React, jQuery, D3, FontAwesome, etc.).
 its original state.
 - Run `yarn start` to start watching your files for changes.
 - Then, in another terminal tab navigate into the `local_test` directory and run
-`yarn serve-https-local`.
+`local-server`.
```
[the remark this answers is not in the document]
Comment (ValeraS) [reply CONTRIBUTING.md]: @tchaffee `live-server` is in our dev dependencies. You can find executable file in the `node_modules/.bin/` folder in the repo.

[the remark this answers is not in the document]
Comment (ValeraS) [reply CONTRIBUTING.md]: @tchaffee I'll redo now

**Comment by ValeraS** [comment]
@tchaffee It's done. And can you look at this https://github.com/freeCodeCamp/testable-projects-fcc/issues/512#issuecomment-415656852?

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tchaffee) [reply CONTRIBUTING.md]: @ValeraS sorry to be so picky about this, but if someone asks me in the future where that command is, I going to forget and have to dig for it. Could you just change this line to the following?

```
 yarn live-server
```

yarn is smart enough to find it in `node_modules/.bin/` and just execute it.

[the remark this answers is not in the document]
Comment (tchaffee) [reply CONTRIBUTING.md]: Actually, I take that back. I'm not sure we even need that command. That command as-is just serves up the current directory. For the local testing I think we need two things. @tbushman please help us out here. Am I right?

1. bundle.js needs to be served and available locally. I think we can use `yarn develop` for that once this PR is merged.
1. The `local_test/` dir needs to be served up and available locally, and hopefully watched with changes hot-reloaded.

I'm not sure what to use for item 2, but I don't think it's just `yarn live-server` on its own. Perhaps `yarn live-server local_test` would work? I'm not sure if that watches for changes though or if that extra command is even necessary. Will `yarn develop` also serve and watch what's in local_test? @tbushman can you clone this branch and test it?

[the remark this answers is not in the document]
Comment (ValeraS) [reply CONTRIBUTING.md]: @tchaffee
doc says to navigate into the `local_test` and run `yarn live-server`

1. `yarn start` watches for changes and recompile bundle.js into the `local-test/js` folder
1. `yarn live-server` serves files from the `local-test` folder and watches with changes hot-reload

[the remark this answers is not in the document]
Comment (tchaffee) [reply CONTRIBUTING.md]: @ValeraS Thanks for pointing that out. Makes sense now.

**Pull request merged**